rubrik_cdm/data_management.py

```diff
@@ -1542,3 +1542,31 @@ def sql_instant_recovery(self, db_name, date, time, sql_instance=None, sql_host=
             self.log("sql_instant_recovery: Performing instant recovery of {} to recovery_point {} at {}.".format(db_name, date, time))
 
             return self.post('v1', '/mssql/db/{}/restore'.format(mssql_id), config, timeout)
+
+    def vcenter_refresh_vm(self, vm_name, timeout=15):  # pylint: ignore
+        """Refresh a single vSphere VM metadata.
+
+        Arguments:
+            vm_name {str} -- The name of the vSphere VM.
+
+        Keyword Arguments:
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            no content.
+        """
+
+        self.log("vcenter_refresh_vm: Searching the Rubrik cluster for the vSphere VM '{}'.".format(vm_name))
+        data = self.get('v1', '/vmware/vm?name={}'.format(vm_name), timeout)
```

Comment:
Making use of `get('v1', '/vmware/vm?name={}` in order to retrieve the VM ID and vcenter ID in a single call.